scripts/image-builder/build.sh

```diff
@@ -258,40 +262,40 @@ createrepo openshift-local >/dev/null
 if ${PROMETHEUS} ; then
     owner=ncabatoff
     repo=process-exporter
-    version=$(curl -s https://api.github.com/repos/${owner}/${repo}/releases/latest | jq -r '.name')
-    url=https://github.com/${owner}/${repo}/releases/download/${version}/
-    file=${repo}_${version#v}_linux_amd64.rpm
+    version="$(curl -s https://api.github.com/repos/${owner}/${repo}/releases/latest | jq -r '.name')"
+    url="https://github.com/${owner}/${repo}/releases/download/${version}/"
+    file="${repo}_${version#v}_linux_amd64.rpm"
```

Comment:
These would better declared local

Reply:
Had to move this to it's own function to be able to make them local, but that's better so there's less chance of accidental use.